src/main/groovy/com/morpheusdata/digitalocean/cloud/sync/VirtualMachineSync.groovy

```diff
@@ -259,7 +275,7 @@ class VirtualMachineSync {
                 maxMemory        : cloudItem.memory * ComputeUtility.ONE_MEGABYTE,
                 maxCores         : (cloudItem.vcpus?.toLong() ?: 0) * (cloudItem.disk?.toLong() ?: 0),
                 coresPerSocket   : 1l,
-                osType           : 'unknown',
+                osType           : cloudItem.image ? cloudItem.image?.distribution : 'unknown',
```

Comment:
osType should just be "linux" or "windows":
```
def osType = cloudItem.image?.distribution?.toLowerCase()?.contains('windows') ? 'windows' : 'linux'
...
[
    osType: osType
]
```

Reply:
@ddeyoung changes done